src/nus/cs2103/addressbook/AddressBook.java

```diff
@@ -557,53 +644,58 @@ private static void updateLatestViewedPersonListing(List<String[]> newListing) {
      */
 
     /**
-     * Create storage file if it does not exist. Shows feedback to user.
+     * Creates storage file if it does not exist. Shows feedback to user.
      *
      * @param filePath file to create if not present
      */
-    private static void tryToCreateFileIfMissing(String filePath) {
+    private static void createFileIfMissing(String filePath) {
         final File storageFile = new File(filePath);
+        if (storageFile.exists()) {
+            return;
+        }
+
+        showToUser(String.format(MESSAGE_ERROR_MISSING_STORAGE_FILE, filePath));
+
         try {
-            if (storageFile.createNewFile()) {
-                showToUser(String.format(MESSAGE_ERROR_MISSING_STORAGE_FILE, filePath));
-                showToUser(String.format(MESSAGE_STORAGE_FILE_CREATED, filePath));
-            }
```

Comment:
Forgot to comment why I changed this. I try to follow the command-query separation principle when I can. When a command operation (i.e. not a query operation) returns the result as the return value, using that value in the code makes the call site code hard to follow. e.g. `if (storageFile.createNewFile())` is not very intuitive. Even Java doesn't follow that principle, so it can't be all that bad. But whenever we can, we should try to follow it.